profile/README.md

```diff
@@ -0,0 +1,46 @@
+## Hello 👋
+
+![An illustration showing the Strapi logo and some confetti.](./banner.png)
+
+Strapi's story started on Github on October 1, 2015. Since this first commit, +16,500 others followed, more than 5,000 issues have been opened and more than 4,800 pull requests were created 🤯 The majority of [milestones associated pull requests](https://github.com/strapi/strapi/milestones) come from the Strapi community and we can't be more proud of it 🙏
+
+We strive every day to create the best Open-Source and Self-Hosted Headless CMS to allow Developers as well as Content Creators to Unleash Content 💪
+
+[See our roadmap](https://portal.productboard.com/strapi/1-roadmap/tabs/2-under-consideration)
+
+### 🚀 Strapi
+
+Design APIs fast, manage content easily.
+
+- [Strapi](https://github.com/strapi/strapi) 
```

Comment:
What do you think about adding a small description of what each of those repos are? Or is it already clear enough?